pkg/cmd/server/origin/asset.go

```diff
@@ -218,6 +220,20 @@ func (c *AssetConfig) addHandlers(mux *http.ServeMux) error {
 	}
 	mux.Handle(configPath, assets.GzipHandler(configHandler))
 
+	// Server version
+	kVersionInfo := kversion.Get()
+	oVersionInfo := oversion.Get()
+	versionInfo := assets.WebConsoleVersion{
+		KubernetesVersionInfo: kVersionInfo,
+		OpenShiftVersionInfo:  oVersionInfo,
+	}
+	versionHandler, err := assets.VersionHandler(versionInfo)
+	if err != nil {
+		return err
+	}
+	versionPath := path.Join(publicURL.Path, "version.js")
```

Comment:
does this have to be served in a separate file? don't really want lots of individual requests for config data